src/instrument.js

```diff
@@ -70,7 +70,7 @@ function computeNextEntry(reducer, action, state, error) {
   let nextState = state;
   let nextError;
   try {
-    nextState = reducer(state, action);
+    nextState = reducer(state, { ...action, replaying });
```

Comment:
I've been thinking about serializability. However, it **does** make sense to ignore that field when actions are serialized anyway.

Agreed that the naming clash can potentially be an issue, will re-work this.